### PR TITLE
feat(CC): add central network connection bandwidth associate resource

### PR DESCRIPTION
**Pull request by profoundwu**
**What this PR does / why we need it**:
add central network connection bandwidth associate resource
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add central network connection bandwidth associate resource
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/cc" TESTARGS="-run TestAccCentralNetworkConnectionBandwidthAssociate_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cc -v -run TestAccCentralNetworkConnectionBandwidthAssociate_basic -timeout 360m -parallel 4
=== RUN   TestAccCentralNetworkConnectionBandwidthAssociate_basic
=== PAUSE TestAccCentralNetworkConnectionBandwidthAssociate_basic
=== CONT  TestAccCentralNetworkConnectionBandwidthAssociate_basic
--- PASS: TestAccCentralNetworkConnectionBandwidthAssociate_basic (373.35s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cc        373.448s

```

### CODE REVIEW

File: huaweicloud/services/cc/resource_huaweicloud_cc_central_network_connection_bandwidth_associate.go [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+			"bandwidth_type": "TestBandwidth",
+		},
+	}
+}
+
+func resourceCentralNetworkConnectionBandwidthAssociateImportState(_ context.Context,
+	d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData, error) {
+	parts := strings.SplitN(d.Id(), "/", 2)
+	if len(parts) != 2 {
+		return nil, fmt.Errorf("invalid format specified for import id, must be <central_network_id>/<id>")
```
Comment (houpeng80) [inline huaweicloud/services/cc/resource_huaweicloud_cc_central_network_connection_bandwidth_associate.go]:
```suggestion
		return nil, fmt.Errorf("invalid format specified for import id, must be <central_network_id>/<connection_id>")
```

Comment (profoundwu) [reply huaweicloud/services/cc/resource_huaweicloud_cc_central_network_connection_bandwidth_associate.go]: fixed

File: huaweicloud/services/cc/resource_huaweicloud_cc_central_network_connection_bandwidth_associate.go [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+	return map[string]interface{}{
+		"central_network_connection": map[string]interface{}{
+			"bandwidth_type": "TestBandwidth",
+		},
+	}
+}
+
+func resourceCentralNetworkConnectionBandwidthAssociateImportState(_ context.Context,
+	d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData, error) {
+	parts := strings.SplitN(d.Id(), "/", 2)
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/cc/resource_huaweicloud_cc_central_network_connection_bandwidth_associate.go]:
```suggestion
	parts := strings.Split(d.Id(), "/")
```

**Pull request merged**